locals.tf

```diff
@@ -4,6 +4,14 @@ locals {
     (contains(["rapid", "stable", "node-image"], var.automatic_upgrade_channel) && var.kubernetes_version == null && var.default_node_pool.orchestrator_version == null)
   )
   dns_prefix = coalesce(var.dns_prefix, random_string.dns_prefix.result)
+  kube_admin_enabled = (
+    !var.local_account_disabled
```

Comment:
Could we avoid double negative statement here?

Reply:
I could do `var.local_account_disabled == false`. The property in the AKS resource in TF is `local_account_disabled` so I'd like to keep the same name